src/components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Wrapper from './Wrapper';
+import HiddenCheckBox from './HiddenCheckBox';
+import CheckMark from './CheckMark';
+import CheckBoxLabel from './CheckBoxLabel';
+
+function CheckBox({ labelText, ...props }) {
```

Comment:
Should be rather **Checkbox** (without capital B).
*labelText* prop could be just **label**.
For accessibility purposes whole checkbox should be wrapped by \<label> element (so clicking text will also check/uncheck checkbox):
```
<label>
  <input type="checkbox" />
  Some text
</label>
```
Would also pass **checked** prop explicitly: `{ label, checked, ...props }` and show / hide tick based on it: `checked && <Tick />` - would make CSS cleaner.